packages/manager/apps/container/src/components/LiveChat.tsx

```diff
@@ -10,11 +12,32 @@ interface LiveChatProps {
   onReduce?: () => void;
 }
 
+const getLevelCustomer = (level: string) => {
+  switch (level) {
+    case 'standard':
+      return 'STD';
+    case 'premium':
+      return 'PRM';
+    case 'business':
+      return 'BUS';
+    case 'enterprise':
+      return 'ENT';
+    case 'premium-accredited':
+        return 'PRP';
+    default:
+      return 'STD';
+  }
+};
+
 const LiveChat: React.FC<ComponentProps<LiveChatProps>> = (
   props: LiveChatProps,
 ): JSX.Element => {
   const { language, open, reduced, subsidiary, ...rest } = props;
-  const url = `https://chat.ovh.com/system/templates/liveChat-manager/STD/${subsidiary}_${language}/docs/index2.html`;
+
+  const level = useApplication().environment?.user?.supportLevel.level;
```

Comment:
Why not providing `supportLevel` through props (as it was done for others props cf https://github.com/ovh/manager/blob/master/packages/manager/apps/container/src/container/index.tsx#L72-L80 ) 
to avoid to link this component to the context ?

Reply:
It's cleaned up